nix/scriptworker.yaml

```diff
@@ -0,0 +1,61 @@
+provisioner_id: scriptworker-prov-v1
```

Comment:
maybe this file should be in `nix/configs` folder

Reply:
Easy!